src/pages/NewUsers.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ProfileStepper from '../components/ProfileStepper';
+
+function NewUsers() {
+  return (
+    <div className="stepperStyle">
```

Comment:
Use a `<Container>` element instead of a `<div>`

Reply:
This page is now called `GetStarted` and I changed to the `<Container>`

Reply:
`<Container> ` got changed now